CRM/Contact/BAO/Group.php

```diff
@@ -442,7 +450,17 @@ public static function create(&$params) {
         $params['parents'] = [$domainGroupID];
       }
 
-      // FIXME: Only allows adding parents, cannot remove them
+      // first deal with removed parents
+      if (!empty($params['id']) && !empty($currentParentGroupIDs)) {
```

Comment:
In the api, the expectation is that if you omit a param then it will not be changed, however the logic in this block is that if you omit `$params['parents']` it will be deleted!
I think this is the fix:
```suggestion
      if (array_key_exists('parents', $params) && !empty($currentParentGroupIDs)) {
```

Reply:
Ah, yes, and `$params['id']` is implicit if we have content in `$currentParentGroupIDs`. I often prefer explicit but in that case, it's really not necessary.

Reply:
Hmm, sorry, I haven't tested this thoroughly and it doesn't work because parents is initialized to NULL at the beginning of the method... Opened another [PR-26260](https://github.com/civicrm/civicrm-core/pull/26260) to fix this